frontend/integration-tests/views/secrets.view.ts

```diff
@@ -0,0 +1,12 @@
+import { $, $$ } from 'protractor';
+
+export const secretNameInput = $('#secret-name');
+export const revealValuesLink = $('#toggle-secret');
```

Comment:
If you use `by.partialButtonText`, then you don't need to add the extra `id` to the element just for tests.

```js
element(by.partialButtonText('Reveal Values'));
```